plugins/field-slider/test/field_slider_test_helpers.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const assert = require('chai').assert;
+const assertFieldValue = require('../test/field_test_helpers').assertFieldValue;
```

Comment:
It might be clearer, and useful if you add more imports here from the same file to use object destructuring. ie: 
```
const {assertFieldValue} = require('../test/field_test_helpers')
```

Reply:
I didn't realize that syntax was valid. I'll update.